bundlewatch.config.js

```diff
@@ -12,7 +12,7 @@ const getMaxSizeForPackage = packageLocation => {
 
 const getChangedPackageLocations = () => {
     let outputPackages;
-    let command = process.env.CIRCLE_BRANCH === 'master' || process.env.RUN_ALL === 'true' ? "npx turbo run build --dry=json" : "npx turbo run build --filter=[origin/master] --dry=json" 
+    let command = process.env.CIRCLE_BRANCH === 'master' || process.env.RUN_ALL === 'true' ? "npx turbo run build --dry=json" : "npx turbo run build --filter=...[origin/master] --dry=json" 
```

Comment:
If master, serve all components in storybook.

Otherwise return a JSON object of changed components and their dependants to be served by Storybook.

Reply:
Worth having these as code comments at all?

Reply:
Done!